src/patterns/components/pagination/collection.yaml

```diff
@@ -4,8 +4,10 @@ description: |
   It serves as navigation and informs users where they currently are and where they
   can go in the future.
 information:
-  - Spark-Core Pagination JS will take care of adding in the correct aria-label
-    to each link on first page load and when the items are tapped or clicked.
+  - As a general guideline, you should use the "default" variant when the pager has 3 or fewer pages,
+    and you should use the "long" variant when the pager has more than 3 pages.
+  - The React component will automatically switch between the "default" and "long" variants
```

Comment:
We'll want any React information under the React section, not in the main area. That away, if someone doesn't have "React" selected in the top page level code switcher, they don't see things related to React.